.github/pull_request_template.md

```diff
@@ -49,7 +55,7 @@
 <!-- Add description of the change -->
 ...
 
-<!-- Add all issues that are fixed by this PR -->
+<!-- Add all issues that are fixed by this PR, use "Part of" if you want to keep issues open. -->
```

Comment:
To make it even more clear
```suggestion
<!-- Add all issues that are fixed by this PR, use "Part of" instead of "Fixes" if you want to keep issues open. -->
```